BlockSettleSigner/EasyEncValidator.cpp

```diff
@@ -25,19 +25,19 @@ QValidator::State EasyEncValidator::validate(QString &input, int &pos) const
    QString tmpInput = input.trimmed().toLower().remove(QChar::Space);
    QString newInput;
    int newPos = 0;
-   const auto &allowedChars = codec_->allowedChars();
+//   const auto &allowedChars = codec_->allowedChars();
    for (int i = 0; i < tmpInput.length(); i++) {
       if ( i && i % wordSize_ == 0) {
          newInput.append(QChar::Space);
          ++newPos;
       }
       const char c = tmpInput.at(i).toLatin1();
-      if ((allowedChars.find(c) == allowedChars.end())) {
+/*      if ((allowedChars.find(c) == allowedChars.end())) {
          input = newInput;
          pos = newPos;
          setStatusMsg(invalidMsgTmpl_.arg(name_));
          return QValidator::State::Invalid;
-      }
+      }*/
```

Comment:
Can you please remove input validation, if it is not used?

Reply:
Input validation is used, I just only commented out the validation of easy16 chars which wasn't working for some reason with new code. Left it commented (not removed) to return to it later after the list will be available from Armory code.

Reply:
Can you add some text marker like TODO, XXX or anything else that will explain why code is left and what should be done and how and when.

Reply:
added